libcrux-ml-kem/src/types.rs

```diff
@@ -189,3 +189,29 @@ impl<const PRIVATE_KEY_SIZE: usize, const PUBLIC_KEY_SIZE: usize>
         (self.sk, self.pk)
     }
 }
+
+#[allow(non_snake_case)]
+/// Types for the unpacked API.
+pub mod unpacked {
+    use crate::{polynomial::PolynomialRingElement, vector::traits::VectorType};
+
+    /// An unpacked ML-KEM Private Key
+    pub struct MlKemPrivateKeyUnpacked<const K: usize, Vector: VectorType> {
+        pub(crate) secret_as_ntt: [PolynomialRingElement<Vector>; K],
+    }
+
+    /// An unpacked ML-KEM Public Key
+    pub struct MlKemPublicKeyUnpacked<const K: usize, Vector: VectorType> {
+        pub(crate) t_as_ntt: [PolynomialRingElement<Vector>; K],
+        pub(crate) seed_for_A: [u8; 32],
+        pub(crate) A_transpose: [[PolynomialRingElement<Vector>; K]; K],
+    }
+
+    /// An unpacked ML-KEM KeyPair
+    pub struct MlKemKeyPairUnpacked<const K: usize, Vector: VectorType> {
+        pub private_key: MlKemPrivateKeyUnpacked<K, Vector>,
```

Comment:
Should these really be public?

Reply:
I made them public (only) for use in tests. If there is a different idiomatic way of doing that we can do that as well.